std/traits.d

```diff
@@ -4758,8 +4758,9 @@ if (isTypeTuple!U)
         enum maxAlignment = U[0].alignof;
     else
     {
-        import std.algorithm.comparison : max;
-        enum maxAlignment = max(staticMap!(.maxAlignment, U));
+        enum a = maxAlignment!(U[0 .. ($+1)/2]);
+        enum b = maxAlignment!(U[($+1)/2 .. $]);
+        enum maxAlignment = a > b ? a : b;
```

Comment:
See how I did a logN expansion rather than a linear recursive expansion? I think that probably does the job, no?

Reply:
Also, compared to instantiating `max`, this is practically nothing.

Reply:
...but I was also pondering whether I should use CTFE rather than a template recursion.
I don't know enough about the cost/benefit tradeoff on that front. At what point does CTFE become profitable?

Reply:
Probably always, @UplinkCoder ?

Reply:
Also, truth is that `U.length` almost always `== 2`

Reply:
In which case just do a `static if (U.length == 2)` branch ;)

Reply:
did